tests/unit/utils/test_deploy_utils.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
```

Comment:
Your class test is not in the good path. you added it in the utils test but you're testing command deploy and not utils.
You can move it to tests/unit/commands/image/test_deploy.py